app/http.php

```diff
@@ -408,15 +409,15 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
     ->param('command', '', new Text(1024), 'Commands to run after container is created. Maximum of 100 commands are allowed, each 1024 characters long.', true)
     ->param('timeout', 600, new Integer(), 'Commands execution time in seconds.', true)
     ->param('remove', false, new Boolean(), 'Remove a runtime after execution.', true)
-    ->param('cpus', 1, new Integer(), 'Container CPU.', true)
+    ->param('cpus', 1, new FloatValidator(), 'Container CPU.', true)
```

Comment:
```suggestion
    ->param('cpus', 1, new FloatValidator(true), 'Container CPU.', true)
```
Not sure why it wasn't loosy before. It was a bug.